### ISSUE TITLE
Remove ispot.tv (false positive)

### ISSUE THREAD

**Issue by llacb47** [lightswitch05/hosts#138]
Hi,

Please remove all `ispot.tv` domains, the site is "The IMDB of Commercials" [according to TechCrunch](https://techcrunch.com/2012/12/04/ispot-tv-launch/) and has a library of TV advertising spots. I don't think they do any web tracking or analytics, except for this domain `pt.ispot.tv` ([source](https://publicwww.com/websites/%22pt.ispot.tv%22/)), which is already included in other lists, for example StevenBlack's unified hostfile.

Example URL: `https://www.ispot.tv/ad/ovRb/expedia-pet-friendly-spa-weekend-song-by-clarence-frogman-henry`

**Comment by lightswitch05** [comment]
Alright, before I get started on this one, I just _have_ to point out that commercials are advertisements - which is literally the point of this list. But I do recognize that for some unknown reason there are people out there that purposefully subject themselves to advertisements in commercial form. Also, I have made an effort to allow user-initiated advertising/tracking. For example, I try not to block those `click.` redirect URLs that don't do tracking unless someone chooses to click on them. Alright, time to dig into this on: 

* It looks like I had a very poor commit message when I added the root domain: https://github.com/lightswitch05/hosts/commit/ba8124b5
* Looks like I added `pi.ispot.tv` because it was being used by the Hulu app on Roku: https://github.com/lightswitch05/hosts/commit/025913d3
* `pt.ispot.tv` would have came from pihole logs again, probably through Hulu again: https://github.com/lightswitch05/hosts/commit/e0c89654
* `staging-pi.ispot.tv` another poor commit message: https://github.com/lightswitch05/hosts/commit/e571b285
* `staging-ps.ispot.tv` another poor commit message: https://github.com/lightswitch05/hosts/commit/6f14e4f4

Ok, so I don't have a lot of history here to go off of. Very likely I saw the domain on the roku, linked it to my Hulu app, did a quick duckduckgo search and saw this:

> iSpot is the leader in TV Ad measurement and TV Attribution. Disruptive brands trust iSpot for real-time and always-on media measurement, competitive intelligence, attention analytics, OTT measurement, TV Attribution, marketing stack integrations, exposure level data, and other advanced analytics.

On their website:

![image](https://user-images.githubusercontent.com/598803/75837677-8e779a00-5d93-11ea-81d3-93e8009805f5.png)

Privacy Policy:

> We provide analytics services ("iSpot Services") that allow advertisers, web/app developers, and third party ad platforms (collectively referred to as “Clients”) to better understand the effectiveness of their content, including online, OTT, and linear advertising. We help Clients understand, for example, whether their ads tend to be viewed or skipped by consumers or how well their ads drive consumers into their funnel. In providing iSpot Services, we may work with Clients who collect and provide us with information collected from consumers’ browsers or devices.

In summary, I understand that they have a large database of commercials available for people to watch. I also understand that some people want to make use of that service. However, they very clearly are doing advertising and tracking across multiple services and devices beyond just their own website - making them a perfect candidate for the list. I think this particular website is a perfect example of why someone might want to `opt-in` to this tracking and advertising by choosing to whitelist them on their own.

**Comment by lightswitch05** [comment]
Oh I forgot to add, thank you for opening this ticket so that we could have this discussion. Although I did not agree with you on this particular matter, I really do appreciate the feedback. If nothing else, it gives me a reason to research why a particular host is blocked and how it fits into the goals of the project.

**Comment by llacb47** [comment]
Could `www.ispot.tv` be whitelisted? (not any of the other domains) This would allow the site to work but still block all third-party tracking.

**Comment by lightswitch05** [comment]
Yeah, that sounds like a totally reasonable request. I'll do that right now

**Comment by lightswitch05** [comment]
`www.ispot.tv` whitelisted by https://github.com/lightswitch05/hosts/commit/99c096ef9a7916616a8944eb43d531a68f5f13c8
